src/matrixroomhandler.ts

```diff
@@ -271,8 +271,25 @@ export class MatrixRoomHandler {
                 });
             }
 
-            const guildId = args[0];
-            const channelId = args[1];
+            let guildId: string;
+            let channelId: string;
+
+            const AMOUNT_OF_IDS_DISCORD_IDENTIFIES_ROOMS_BY = 2;
+
+            if (args.length === AMOUNT_OF_IDS_DISCORD_IDENTIFIES_ROOMS_BY) { // "x y" syntax
+                guildId = args[0];
+                channelId = args[1];
+            } else if (args.length === 1 && args[0].includes("/")) { // "x/y" syntax
+                const split = args[0].split("/");
+                guildId = split[0];
+                channelId = split[1];
+            } else {
+                return this.bridge.getIntent().sendMessage(event.room_id, {
+                    body: "Invalid syntax",
```

Comment:
Maybe a bit more descriptive output, like explaining what the valid syntax is?

Reply:
How do you handle formatted body in the tests?

Reply:
I've changed the error message to "Invalid syntax: See `!discord help`"